### PR TITLE
Create auto-installation for Leap 15.4 in Yam Support job group in O3

### PR DESCRIPTION
**Pull request by lemon-suse**
We need create auto-installation for Leap 15.4 in Yam Support job group in O3.
Related ticket: https://progress.opensuse.org/issues/122173
VR:
    Created job for the qcow of leap 15.4: https://openqa.opensuse.org/tests/3100615#
    VR for failed job: https://openqa.opensuse.org/tests/3101018#step/opensuse_welcome/3  failed for needle issue now, but can't save a new needle for git issue.

### CODE REVIEW

File: job_groups/support_images.yaml [hunk 1]
```diff
@@ -26,3 +26,9 @@ scenarios:
             PUBLISH_HDD_1: opensuse-updated-%VERSION%-%ARCH%-%DESKTOP%-%BUILD%.qcow2
             OS_TEST_ISSUES: ""
             QA_HEAD_REPO: ""
+      - create_hdd_leap_gnome_autoyast_dev:
```
Comment (jknphy) [inline job_groups/support_images.yaml:29]: I don't know what you trying to accomplish here inheriting from a new test suite that we create directly in the web ui (or db).  Ideally we should not need that, just `testsuite: null` with a good description of this new test suite.
I mean we can use archor and reference in yaml if we repeat stuff among architectures in the future, but for now we could just write all here.
We can also remove the test suite above, which is just a dummy test.
A minor thing, doesn't need to be named with `_dev`.
We normally don't clear `QA_HEAD_REPO` for example in normal TW, let's leave, I don't know what is its use but the one that we could clear to not have maintenance settings there is OS_TEST_TEMPLATE

Comment (lemon-suse) [reply job_groups/support_images.yaml:29]: Thanks, updated the test suite in yaml. VR updated also.

File: job_groups/support_images.yaml [hunk 2]
```diff
@@ -26,3 +26,15 @@ scenarios:
             PUBLISH_HDD_1: opensuse-updated-%VERSION%-%ARCH%-%DESKTOP%-%BUILD%.qcow2
             OS_TEST_ISSUES: ""
             QA_HEAD_REPO: ""
+      - create_hdd_leap_gnome_autoyast:
```
Comment (jknphy) [inline job_groups/support_images.yaml:29]: please remove dummy test suite above we introduced in previous task.

File: job_groups/support_images.yaml [hunk 3]
```diff
@@ -26,3 +26,15 @@ scenarios:
...
+          testsuite: null
+          settings:
+            DESKTOP: gnome
+            HDDSIZEGB: "30"
+            INSTALLATION_VALIDATION: console/autoyast_smoke
```
Comment (jknphy) [inline job_groups/support_images.yaml:34]: afir we might not need:
`INSTALLATION_VALIDATION` and `INSTALLONLY`, could you please check it?

**Review by jknphy — COMMENTED** [review]
do we have a passing job for the failing one? I didn't find it in your first comment.

File: job_groups/support_images.yaml (hunk 2 above)
Comment (lemon-suse) [reply job_groups/support_images.yaml:29]: I can't remove it cause someone use it in https://openqa.opensuse.org/admin/job_templates/98 
  x86_64:
    opensuse-15.4-DVD-Updates-x86_64:
      - autoyast_leap_videomode_text
      - create_hdd_leap_gnome_autoyast

Comment (lemon-suse) [reply job_groups/support_images.yaml:29]: How about to change the name in yaml? Any suggestion?

Comment (jknphy) [reply job_groups/support_images.yaml:29]: ahm ok, so it is not dummy test test suite, but we (ourselves) added there as dummy, just remove it from this file.

File: job_groups/support_images.yaml (hunk 3 above)
Comment (lemon-suse) [reply job_groups/support_images.yaml:34]: > do we have a passing job for the failing one? I didn't find it in your first comment.

Here is a passed one https://openqa.opensuse.org/tests/3016329#step/reboot_and_install/6

File: job_groups/support_images.yaml (hunk 2 above)
Comment (lemon-suse) [reply job_groups/support_images.yaml:29]: So we can close this PR I think.

File: job_groups/support_images.yaml (hunk 3 above)
Comment (jknphy) [reply job_groups/support_images.yaml:34]: but that is not using Leap 15.4, how is that?

File: job_groups/support_images.yaml (hunk 2 above)
Comment (jknphy) [reply job_groups/support_images.yaml:29]: PR? what do you mean, I was just suggesting to remove
`create_hdd_leap_textmode_autoyast_dev` from this file, but your new test suite is fine.

Comment (lemon-suse) [reply job_groups/support_images.yaml]: Clear now, updated yaml file and VRs.

File: job_groups/support_images.yaml (hunk 3 above)
Comment (lemon-suse) [reply job_groups/support_images.yaml]: Updated.

File: job_groups/support_images.yaml [hunk 4]
```diff
@@ -20,9 +20,13 @@ products:
...
           settings:
-            PUBLISH_HDD_1: opensuse-updated-%VERSION%-%ARCH%-%DESKTOP%-%BUILD%.qcow2
+            DESKTOP: gnome
+            HDDSIZEGB: "30"
             OS_TEST_ISSUES: ""
-            QA_HEAD_REPO: ""
+            PUBLISH_HDD_1: opensuse-updated-%VERSION%-%ARCH%-%DESKTOP%-%BUILD%.qcow2
+            PUBLISH_PFLASH_VARS: "%DISTRI%-%VERSION%-%ARCH%-%BUILD%-%DESKTOP%_ay@%MACHINE%-uefi-vars.qcow2"
+            QEMURAM: "2048"
+            YAML_SCHEDULE: schedule/functional/autoyast_create_hdd/autoyast_create_hdd_leap_update.yaml
```
Comment (jknphy) [inline job_groups/support_images.yaml:32]: shouldn't be named the schedule as `*_updated` ?

File: job_groups/support_images.yaml (hunk 3 above)
Comment (jknphy) [reply job_groups/support_images.yaml]: could you please update the first comment in the PR with current VR?

Comment (lemon-suse) [reply job_groups/support_images.yaml]: This already updated in PR description: https://openqa.opensuse.org/tests/3100615#

**Pull request merged**